### PR TITLE
feat: Android Build Support for ChatApp

### PR DESCRIPTION
**Pull request by guitarrapc**
## TL;DR

Android Build Support for ChatApp.

## Summary

* Build on CircleCI.
* Android IL2CPP Build on macOS
* Require UserName/ Password / Serial to activate Unity License on CirclecI macOS. (you cannot use Offline license as Docker do)
  * UNITY_PASSWORD
  * UNITY_USERNAME
  * UNITY_SERIAL
image

## TODO

* [x] Android Build Support on Unity
* [x] Unity Batch Build
* [x] Unity Licesing on CircleCI
  * PLEASE ADD USER/PASS/SERIAL on CircleCI Environment Variables
* [x] Android SDK/NDK support
* [x] Artifact support
* [x] Modify .gitignore to allow upm package manifests.json

## Artifacts

image

### CODE REVIEW

**Comment by guitarrapc** [comment]
1st time build will take 21min, because of unity download.

image
